router.go

```diff
@@ -238,9 +251,19 @@ func StringErrorResponse(err string) APIResponse {
 	return APIResponse{Error: err, Status: "ERROR", Data: nil}
 }
 
-// ErrorResponse constructs error response from the API
+// ErrorResponse constructs error response from the API with error level log
 func ErrorResponse(err error) APIResponse {
-	return APIResponse{Error: err.Error(), Status: "ERROR", Data: nil}
+	return APIResponse{Error: err.Error(), Status: "ERROR", Data: nil, Type: LogTypeError}
+}
+
+// InfoResponse constructs error response from the API with info level log
+func InfoResponse(err error) APIResponse {
```

Comment:
This should be ErrorInfoResponse() since apart from log level the response is actually an error

Reply:
https://github.com/phil-inc/pweb/pull/27/commits/041f35535759daca50be279de9ab6e108a0e28de